JortPob/Common/Utility.cs

```diff
@@ -129,11 +134,24 @@ public static List<string> CivilizedSplit(string text)
             return recombined;
         }
 
-        public static string[] StringAwareSplit(string text)
+        public static string StringAwareLower(string s)
+        {
+            bool inQuote = false;
+            string text = s;
+            for (int i = 0; i < text.Length; i++)
```

Comment:
might be worth making this just use `StringBuilder` and build up `text` from scratch, same w/ the other `StringAwareX` methods